README.md

```diff
@@ -20,7 +26,19 @@ npm start
 This command starts a local development server and open up a browser window.
 Most changes are reflected live without having to restart the server.
 
-## Release to Github Pages 
+## 👥 How to contact
+
+This project is part of Expedia Group Open Source. If you want to reach out to us, please [raise an issue](https://github.com/ExpediaGroup/open-source-portal/issues)
+on GitHub.
+
+## How to release 
+
+Please see info in [RELEASING.md](./RELEASING.md).
+
+## ✏️ How to contribute
+
+Please see info in [CONTRIBUTING.md](./CONTRIBUTING.md).
+
+## ⚖️ License
 
-Any commit to `main` branch will trigger a GitHub Actions workflow that builds and releases
-the site to Github Pages.
+This repository is licensed under the [Apache License, Version 2.0](./LICENSE).
```

Comment:
@nikos912000 @lasanthak I'd like somehow to give credits / acknowledge we are using the free [Medium Pixel Point widget](https://medium-widget.pixelpoint.io/). Do you know if there is some standard for this kind of things (a file with a naming convention, etc)?

Conversely, I guess I don't need to give credits to all the libraries we depend on in the `package.json`, right?

Thanks!

Reply:
I am not sure if there is a convention.
I have seen a `CREDITS.md` but also a `Credits` section in the readme.

Reply:
I do not think you have to give credit for those projects that are being used, they are public and ppl should know. This feels like a special mention that a particular lib has played a major role.

But we can use a credits file or a section to mention some ppl who have contributed but it is not visible from the project (history, commits, or configs etc.).

In Jarviz, I have given credit to the others who sent PRs in changelog file as well. Not sure if that is necessary as it is obvious from commit history.

Reply:
Thanks both! I've added `CREDITS.md` in f4ee2e8d194012de51197a97b1e11e65f2dc8f4a , let me know if it looks good!